man/loader.conf.xml

```diff
@@ -215,6 +215,26 @@
         Currently, only x86 is supported, where it uses the PC speaker.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>secure-boot-enrollment</term>
+
+        <listitem><para>Danger: this feature might soft-brick your device if used improperly.</para>
+
+	<para>Takes one of <literal>off</literal>, <literal>manual</literal> or <literal>force</literal>.
+	Controls the enrollment of secure boot keys. If set to <literal>no</literal>, no action whatsoever
+	is taken. If set to <literal>manual</literal> (the default) and the UEFI firmware is in setup-mode
+	then entries to manually enroll Secure Boot keys are created in the boot menu. If set to
+	<literal>force</literal>, in addition, if a directory named <filename>/loader/keys/auto</filename>
+	exists on the ESP then the keys in that directory are enrolled automatically.</para>
```

Comment:
this doesn't mention that the menu entries are loaded from `/loader/keys/*`

Reply:
there should also be a comment about these dirs in the "Files" section of systemd-boot.xml